src/navtypes.h

```diff
@@ -12,6 +12,8 @@ struct gpscoords_t {
 	double lat;
 	/** the longitude of the gps coordinate, in degrees */
 	double lon;
+	/** the altitude of the gps coordinate */
```

Comment:
See https://github.com/huskyroboticsteam/ardupilot-driver/pull/2/files and update this comment to specify the unit of data